### ISSUE TITLE
error with 'cacli train' command

### ISSUE THREAD

**Issue by cetienn01** [cloud-annotations/training#91]
everytime i run the 'cacli train' command, i get the following error message:
```
(Using settings from config.yaml)
error Invalid Watson Machine Learning credentials.
```
Do you know how i can access the config.yaml file and change it or replace or even just remove it completely so i can start over?
When i first ran the command, i accidentally missed the part to enter some of the key information like 'instance_id'. but i entered 'access_key_id' and 'secret_access_id'.
I think that's the step that i messed up because i didn't run into any other issues before that.

Thanks,
CE

**Comment by cetienn01** [comment]
I found my config.yaml file, here's what it looks like:
buckets:

```
  training: object.detection.test
  output: object.detection.test
name: object.detection.test
```

**Comment by bourdakos1** [comment]
You can run cacli init

**Comment by bourdakos1** [comment]
Possibly cacli login

**Comment by cetienn01** [comment]
Okay let me try that right now

**Comment by cetienn01** [comment]
so it starts over, but it saved my previous answers and i cannot change them. i can only hit 'enter' to go to next line, and i get the error message:
`warning The provided Watson Machine Learning credentials are invalid.
`
It also asks me for a url, how do i find that?

**Comment by cetienn01** [comment]
Oh wait, what it shows in parentheses is the previous answers, but if i type a new one, it will change to the new entry, Im gonna try that. however, I still don't know where to find the url

**Comment by bourdakos1** [comment]
The url should be somewhere in the credentials

**Comment by cetienn01** [comment]
I think i was entering the wrong Watson Machine Learning credentials.
So i just went back to the credentials we created at the very beginning of the tutorial and found all the information there. What i was entering was based on the cloud storage credentials.
Let me try again with the correct credentials information

**Comment by cetienn01** [comment]
It's working!
So far so good.
After i used 'cacli login' to re-enter all the credentials
then 'cacli train' to begin training the model, everything seems to be working.
No error message.
`success Training run submitted.`

**Comment by aditya9026** [comment]
@cetienn01 update cloud-annotations package or reinstall

**Comment by bachtiarfr** [comment]
Hi guys!

I just had an error like this :
(Using settings from config.yaml)
⠋ Starting training run...(node:2404) UnhandledPromiseRejectionWarning: StatusCodeError: 400 - {"trace":"7559f178b855c6b77cd6c90ca08eb96f","errors":[{"code":"invalid_training_framework","message":"Unsupported training framework specification, tensorflow_1.12","more_info":"http://watson-ml-api.mybluemix.net/"}]}
    at new StatusCodeError (C:\Users\BACHTIARFATURROHIM\AppData\Roaming\npm\node_modules\cloud-annotations\node_modules\request-promise-core\lib\errors.js:32:15)
    at Request.plumbing.callback (C:\Users\BACHTIARFATURROHIM\AppData\Roaming\npm\node_modules\cloud-annotations\node_modules\request-promise-core\lib\plumbing.js:104:33)
    at Request.RP$callback [as _callback] (C:\Users\BACHTIARFATURROHIM\AppData\Roaming\npm\node_modules\cloud-annotations\node_modules\request-promise-core\lib\plumbing.js:46:31)
    at Request.self.callback (C:\Users\BACHTIARFATURROHIM\AppData\Roaming\npm\node_modules\cloud-annotations\node_modules\request\request.js:185:22)
    at Request.emit (events.js:198:13)
    at Request.EventEmitter.emit (domain.js:448:20)
    at Request.<anonymous> (C:\Users\BACHTIARFATURROHIM\AppData\Roaming\npm\node_modules\cloud-annotations\node_modules\request\request.js:1154:10)
    at Request.emit (events.js:198:13)
    at Request.EventEmitter.emit (domain.js:448:20)
    at IncomingMessage.<anonymous> (C:\Users\BACHTIARFATURROHIM\AppData\Roaming\npm\node_modules\cloud-annotations\node_modules\request\request.js:1076:12)
(node:2404) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:2404) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

Can you guys help me to solve this

**Comment by bourdakos1** [comment]
You need to update cacli

**Comment by bachtiarfr** [comment]
How do I update this using terminal, because when I use "$ cacli -h" there's no update command

**Comment by bourdakos1** [comment]
How did you install it/ are you on windows/mac?
